java/ql/src/Security/CWE/CWE-094/InsecureBeanValidation.qhelp

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>Bean validation custom constraint error messages support different types of interpolation, 
```

Comment:
```suggestion
<p>Custom error messages for constraint validators support different types of interpolation, 
```


As above, I've tried to make this easier to read. Feel free to tweak it again if I've misunderstood 😊